docs/SETUP.md

```
@@ -1,30 +1,27 @@
 # Setup
 
-### 🔧 Step 1: Prerequisites
-+ Ensure you have any Python version in the range **[3.11, 3.13]** installed.
-  + If you don't, download one from [here][python-downloads].
-+ Ensure you have the **uv package manager** installed.  
-  + If you don’t, install it by following the steps [here][uv-install].
-+ Clone or download this repository.
+### 📁 Step 1: Get the source code
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)